example.ts

```diff
@@ -10,36 +10,31 @@ class Student {
         this.studentYear = studentYear;
         this.studentGpa = studentGpa;
     }
-
-    static updateGPA(student : Student, new_grade : number) : Student {
-        student.studentGpa = student.studentGpa + new_grade;
-        return student;
-    }
-
+    @DiscreetORM.WriteModifiedToDB(DiscreetORM.SQL_IO)
 	incrementYear() : void {
         // This is javascript big brain time.
         this.studentYear = +this.studentYear + 1 + ''; 
     }
-}
-
-
-class StudentMethods {
-
-    @DiscreetORM.WriteToDB(DiscreetORM.SQL_IO)
+    @DiscreetORM.WriteReturnToDB(DiscreetORM.SQL_IO)
     static updateGPA(student : Student, new_grade : number) : Student {
         student.studentGpa = student.studentGpa + new_grade;
         return student;
     }
 }
+
 let ahad_student = new Student('Ahad', '2021', 0.0);
 let haram_student = new Student('Haram', '2020', 5.0);
 Student.updateGPA(ahad_student, 2.0);
 Student.updateGPA(haram_student, 2.5);
-console.log(ahad_student);
-console.log(haram_student);
 // @ts-ignore
 console.log("Haram's hidden orm id: " + haram_student.discreet_orm_id);
 ahad_student.incrementYear();
+<<<<<<< HEAD
+ahad_student = Student.updateGPA(ahad_student, 1.0);
+ahad_student = Student.updateGPA(ahad_student, 1.5);
+console.log(ahad_student);
+=======
 ahad_student = StudentMethods.updateGPA(ahad_student, 1.0);
+>>>>>>> f23a9b74664023e0b357d3139568d630d7a3e2ea
```

Comment:
Looks like there's a merge conflict here that'll need to be cleared up

Reply:
Yeah this one is easily resolved, it's just because it was behind the old changes
We keep the head